appengine/flexible/storage/main.py

```diff
@@ -57,6 +57,11 @@ def upload():
         content_type=uploaded_file.content_type
     )
 
+    # Make the blob public. This is not necessary if the
+    # entire bucket is public.
+    # See https://cloud.google.com/storage/docs/access-control/making-data-public.
+    blob.make_public()
+
```

Comment:
TODO: check if the tutorial assumes the bucket is public.

Reply:
Yep, [the tutorial](https://cloud.google.com/appengine/docs/flexible/python/using-cloud-storage) says:

> Make the bucket [publicly readable](https://cloud.google.com/storage/docs/access-control#using_permissions_with_acls) so it can serve files: